Dockerfile

```
@@ -1,6 +1,6 @@
 
 # Stage to build Sync Gateway binary
-FROM golang:1.8-stretch as builder
+FROM golang:1.12-stretch as builder
```

We've not tested Sync Gateway with this version of Go yet, both in terms of QE but also performance testing. Let's stick with the latest version we've ran through QE/Perf, defined in `product-config.json`

https://github.com/couchbase/sync_gateway/blob/de03c305e8340a1c04db9216c2c70c1c67590c3f/manifest/product-config.json#L9

```suggestion
FROM golang:1.11.5-stretch as builder
```